integration_test/third_party_apps_data/applications/vault/metadata.yaml

```diff
@@ -139,5 +250,38 @@ configuration_options:
     - name: wildcard_refresh_interval
       default: 60s
       description: The interval at which wildcard file paths in include_paths are refreshed. Given as a time duration, for example 30s or 2m. This property might be useful under high logging throughputs where log files are rotated faster than the default interval.
+  metrics:
+  - type: vault
+    fields: 
+    - name: type
+      default: null
+      description: This value must be vault.
+    - name: endpoint
+      default: localhost:8200
+      description: The hostname:port used by vault
+    - name: token
+      default: localhost:8200
+      description: Token used for authentication.
+    - name: metrics_path
+      default: /v1/sys/metrics
+      description: the path for metrics collection.
+    - name: collection_interval
+      default: 60s
+      description: A time.Duration value, such as 30s or 5m.
+    - name: insecure
+      default: true
+      description: Sets whether or not to use a secure TLS connection. If set to false, then TLS is enabled.
+    - name: insecure_skip_verify
+      default: false
+      description: Sets whether or not to skip verifying the certificate. If insecure is set to true, then the insecure_skip_verify value is not used.
```

Comment:
`insecure_skip_verify`  (or just "the value of this field is not used.")

Reply:
Updated